detroit-labs/typed-things
# material-ui add rowspan

+ Updating our internal typings def for material-ui to v0.16.0 using Typescript 2.1 (NOTE: couldn't go to the very latest from Definitely Typed due to some React incompatibilities: https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/react)
+ New UI table changes require passing `rowSpan` down to `td` and `th` tags, so we need to add that property to the type defs.

### Testing Steps

1. Check out the master front end and middleware branches
2. Start the app locally
3. Update the frontend's top-level `typings.json` file to set the material-ui line to:
```
"material-ui": "github:detroit-labs/typed-things/material-ui/material-ui.d.ts#2af7957",
```
4. Run `yarn typings` for the frontend
5. Observe that there are no error or warnings in the running process
6. Login to the app and poke around.
7. Observe that there are no errors or warnings in the console.
